docs/section21/Rule21-2.md

```diff
@@ -0,0 +1,62 @@
+
+# Boiler - Rule 21-2  
+
+**Rule ID:** 21-2  
+**Rule Description:** Baseline should include all on-site distribution pumps included in the Proposed design.  
+**Rule Assertion:** B-RMR LEN(ASHRAE229:conditioning_component.pump) = P-RMR LEN(ASHRAE229:conditioning_component.pump) for pumps serving purchased hot water or steam loops  
```

Comment:
@xingcx this rule description should be updated to say 'included in the proposed design for purchased HW or steam loops'?

Reply:
Corrected.

Reply:
Corrected.

Reply:
@xingcx ASHRAE 229 data group doesnt have a 'conditioning_component' data group, but just a pump data group. Any reason to incluse the conditioning_component out here 'ASHRAE229:conditioning_component.pump'?